### ISSUE TITLE
Will model parameter changes always contain a parameter with target "parameters.<ID>.value"?

### ISSUE THREAD

**Issue by bilalshaikh42** [reproducible-biomedical-modeling/Biosimulations_utils#21]
When the utils parses a SEDML file with a a list of changes, can it be assumed that the parsed object "Simulation.model_param_changes" will only include those changes whose target is "parameters.<ID>.value" where <ID> is the name of a parameter? Can this object contain other changes as well?

**Comment by jonrkarr** [comment]
No. For example, targets to SBML models can be multiple things:
- Global parameters
- Local parameters of individual reactions
- Initial species counts/concentrations
- (Initial) Compartment volumes
- Assignment rules whose value are constants (e.g., numbers)
- Initial assignments whose value are constants (e.g., numbers) -- sort of special case of assignment rule 
- Additional possible targets for models that use SBML packages
  - FBC package: FBA objective coefficients
  - qual package: initial levels of qualitative variables

While there are many possible types of targets, each target can be identified by a unique XPATH (more generally, a uniquely resolvable path within a model).

I'm not sure how much of the above GillesPy2 supports and how it represents each type of target within its data structures. If this is too complicated, there's a few alternatives:
- (Best alternative): Use an XML library to read models, retrieve the XPaths, and modify their values. This can be implemented modularly from GillesPy2. This would also be applicable to all XML-based models. This would include CellML, NeuroML, SBML, BioNetGen XML, VCell ML, and others.
- Use libSBML to read in models and then modify them. This can be implemented modularly from GillesPy2. However, this would only be applicable to SBML models.
- ~Use Biosimulations_utils to read in models and then modify them~ -- not possible because Biosimulations_utils only reads a small part of models
